app/src/lib/shells/win32.ts

```diff
@@ -185,8 +185,8 @@ async function findHyper(): Promise<string | null> {
     const path = commandPieces
       ? commandPieces[2]
       : localAppData != null
-        ? localAppData.concat('\\hyper\\Hyper.exe')
-        : null // fall back to the launcher in install root
+      ? localAppData.concat('\\hyper\\Hyper.exe')
```

Comment:
I think this is the only real formatting change that bothers me, and maybe this is a chance to avoid the crutch of nested ternary statements.

Reply:
This change was originally made to encourage the use of nested ternaries since they would previously indent for each level of nesting.

Reply:
And I think I had kind of gotten used to that as a code smell, but I'm happy to defer to others how they feel about a) this change, and b) the use of nested ternaries in general.

Reply:
@desktop/comrades @desktop/engineering I think this is the only bit of feedback I had about the PR, but I'm fine with merging as long as the code is still working (if a bit harder to follow)

Reply:
Yea the more nesting kinda encourages me to make sub-functions personally. I think its also easier to read with nesting. Maybe we can turn that off/back?

Reply:
I don't think we can, but @j-f1 can confirm. A suggestion if we still don't like this change - address these areas where we use nested ternaries in a `tech-debt` PR which we can then land and merge into this PR?

Reply:
> I don't think we can

Correct. There isn’t an option for this.

Reply:
Maybe we upstream an option? and pull down that version (maybe thats beyond the scope of this though)?

Reply:
@say25 Prettier has [a policy](https://prettier.io/docs/en/option-philosophy.html) of avoiding options.